python_modules/libraries/dagster-dbt/dagster_dbt/cloud_v2/client.py

```diff
@@ -306,7 +306,7 @@ def poll_run(
         run_id: int,
         poll_interval: Optional[float] = None,
         poll_timeout: Optional[float] = None,
-    ) -> Mapping[str, Any]:
+    ) -> DbtCloudRun:
```

Comment:
For consistency, could we return the run details (mapping) and rebuild the DbtCloudRun where needed? All other methods of the client return either a mapping or a list. 

The reasoning here is that users can interact with the client and get the JSON response, without dealing with our internal representations.

Reply:
that makes sense -- updated!